build.sbt

```diff
@@ -234,6 +234,16 @@ lazy val nvdla = (project in file("generators/nvdla"))
   .settings(libraryDependencies ++= rocketLibDeps.value)
   .settings(commonSettings)
 
+lazy val caliptra_aes = (project in file("generators/caliptra-aes-acc"))
+  .dependsOn(rocketchip, rocc_acc_utils, midasTargetUtils)
+  .settings(libraryDependencies ++= rocketLibDeps.value)
+  .settings(commonSettings)
+
+llazy val rocc_acc_utils = (project in file("generators/rocc-acc-utils"))
```

Comment:
ttypo

Reply:
Also, is there a case for sticking this in testchipip? If rocc accelerators already need to depend on rocket-chip + testchipip, it would reduce some dependency complexity.

Reply:
It makes sense to reduce the dependency complexity... but at the same time I would like to keep `testchipip` as minimal as possible (or at least keep it closer to "test chip stuff" as possible). Right now, `testchipip` isn't enough of a bag of random components to warrant us to put other stuff in there. However, we could make the decision now to make it more of a "bag of ucb-bar stuff".

Reply:
Fair enough